WalletWasabi.Fluent/Views/Wallets/Advanced/WalletCoins/Columns/SelectionColumnView.axaml

```diff
@@ -7,5 +7,5 @@
              x:DataType="walletcoins:WalletCoinViewModel"
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.Wallets.Advanced.WalletCoins.Columns.SelectionColumnView">
-  <CheckBox IsChecked="{Binding IsSelected}" Margin="2 0" HorizontalAlignment="Center" VerticalAlignment="Center" />
+  <CheckBox IsChecked="{Binding IsSelected}" Margin="2 0" HorizontalAlignment="Center" VerticalAlignment="Center" IsEnabled="{Binding !CoinJoinInProgress}" />
```

Comment:
The difference between the enabled and disabled is barely visible. How about hiding the selection completely instead of disabling it?

Current:
![image](https://user-images.githubusercontent.com/16364053/204516078-17e51275-e96f-418d-b055-4c7eae342d45.png)

Idea:
![image](https://user-images.githubusercontent.com/16364053/204515769-4e7e79ee-ec75-4851-a728-46b3771748b2.png)

Reply:
I was thinking about the same, I'd not put somekind of other icon/indicator in place of the box, probably hiding it overall would be the best option.

Reply:
Makes sense. Done in https://github.com/zkSNACKs/WalletWasabi/pull/9657/commits/3a6c2d6a8cb6febe6fb4d5e9ac51447d710f0c4f